exasol_python_test_framework/exatest/__init__.py

```diff
@@ -247,6 +247,7 @@ def _parse_opts(self):
                 logdir='.',
                 loglevel='warning',
                 odbc_log='off',
+                ssl_cert='unverified',
```

Comment:
```suggestion
                tls_cert='unverified',
```

Reply:
While the term "TLS" might be more correct than "SSL", I hesitate to introduce a renaming layer, here. 
The term used in file `odbc.ini` is coined to SSL. So, I fear, we might need to live with that.